docs/en/setup/backend/backend-meter.md

```diff
@@ -0,0 +1,77 @@
+# Meter Receiver
+Meter receiver is accept meter from [meter protocol](https://github.com/apache/skywalking-data-collect-protocol/blob/master/language-agent/Meter.proto) into the [Meter System](./../../concepts-and-designs/meter.md).o
+
+## Module define
+receiver-meter:
+  selector: ${SW_RECEIVER_METER:default}
+  default:
+
+## Configuration file
+Meter receiver is configured via a configuration file. The configuration file defines everything related to receiving 
+ from agents, as well as which rule files to load.
+ 
+OAP can load the configuration at bootstrap. If the new configuration is not well-formed, OAP fails to start up. The files
+are located at `$CLASSPATH/meter-receive-config`.
```

Comment:
I think you should provide Sleuth config in the folder, the example of agent setup, UI template, and full documentation to explain the document.
There is nothing than an example to show how to use this feature.